bundles/org.openhab.binding.daikin/CONTRIBUTING.md

```diff
@@ -0,0 +1,174 @@
+# Contributing to openHAB
```

Comment:
This file shouldn't be part of the PR.

Reply:
Thanks; i have no idea where it came from.